src/upgrader.js

```diff
@@ -244,20 +292,9 @@ class Upgrader {
     }
 
     // Pipe all data through the muxer
-    pipe(muxedConnection, muxer, muxedConnection)
+    pipe(upgradedConn, muxer, upgradedConn)
 
     maConn.timeline.upgraded = Date.now()
```

Comment:
We should just set this right after we created the timeline Proxy, that way we don't need to set it here and in the `!Muxer` if statement above.